ruby/README.md

```diff
@@ -1,6 +1,64 @@
 # Gurmukhi Utils (Ruby)
 
-Utilities library for converting, analyzing, and testing Gurmukhi strings.
+GurmukhiUtils is a utility gem for converting, analyzing, and testing Gurmukhi strings. The library provides various operations like Unicode conversion, ASCII conversion, and more.
```

Comment:
```suggestion
Utilities library for converting, analyzing, and testing Gurmukhi strings.
```

Trying to use a standard line for all the README's so end users can learn to skip reading it basically. If you want to add this info below that would be acceptable